screens/AuthScreen.js

```diff
@@ -201,6 +201,7 @@ const styles = StyleSheet.create({
     borderRadius: 5,
     overflow: 'hidden',
     backgroundColor: Colors.defaultWhite,
+    color: 'black',
```

Comment:
It's better to make it variable.

Reply:
Where do we supposed to see the changes btw, I don't see any difference.

Reply:
I guess he had the issue again that the text was invisible. But I agree with the variable ;-)